.gitlab-ci.yml

```diff
@@ -59,17 +65,22 @@ package-production:
     only:
         - tags
 
-# Not executed, parent job definition for Tomcat deployments
+# Not executed, parent job definition for deployments
 .deploy:
```

Comment:
I think we should separate the `web services` and `clustering pipeline` deployment, that way the deployment of one module will not be affected when the deployment of another module fails.
So we can create `.deploy-ws` and `.deploy-clustering` abstract stages

Reply:
That's a good point. I  split the deploy stages